apps/openmw/mwmechanics/aisequence.cpp

```diff
@@ -366,6 +372,18 @@ void AiSequence::readState(const ESM::AiSequence::AiSequence &sequence)
     if (!sequence.mPackages.empty())
         clear();
 
+    // If there is more than one non-combat, non-pursue package in the list, enable repeating.
+    for (std::vector<ESM::AiSequence::AiPackageContainer>::const_iterator it = sequence.mPackages.begin();
+         it != sequence.mPackages.end(); ++it)
```

Comment:
Would adding up a count every time isActualAiPackage returns true and then checking if it is greater than 1 be an acceptable solution?